app/src/main/java/openfoodfacts/github/scrachx/openfood/views/product/ProductFragment.java

```diff
@@ -43,7 +43,7 @@
 public class ProductFragment extends Fragment implements OnRefreshListener {
 
     private static final int LOGIN_ACTIVITY_REQUEST_CODE = 1;
-    public static State productState;
+    public static State productState;//NOSONAR To be changed ASAP !
```

Comment:
perhaps adding a TODO ?

Reply:
I created an issue about this point:
https://github.com/openfoodfacts/openfoodfacts-androidapp/issues/2703

Reply:
then, LGTM, I'm merging